app/brave_settings_strings.grdp

```diff
@@ -150,6 +150,9 @@
   <message name="IDS_SETTINGS_APPEARANCE_SETTINGS_TABS_SHOW_VERTICAL_TABS" desc="The label for showing tabs vertically">
     Use vertical tabs
   </message>
+  <message name="IDS_SETTINGS_APPEARANCE_SETTINGS_TABS_USE_TREE_TABS" desc="The label for using tree tabs" formatter_data="webui=BraveSettings">
+    Use tree tabs
```

Comment:
@sangwoo108 could you please add a screenshot to the PR. Thanks!

Reply:
Sure! Done.